WordPress/src/main/AndroidManifest.xml

```diff
@@ -107,6 +107,11 @@
         </activity>
 
         <activity
+            android:name=".ui.accounts.LoginActivity"
```

Comment:
Yes, `SignInActivity` will be superseded and eventually dropped along with the feature flag itself.